package.json

```diff
@@ -28,9 +28,9 @@
     "check": "run-s check:*",
     "check:workspaces": "npm run check --workspaces --if-present",
     "check:ws": "[ $(ls -1 ./packages|wc -l) -eq $(cat package.json | jq '.workspaces | length') ] || exit 1 # check no packages left behind",
-    "test": "NODE_ENV=test npm run test --workspaces --if-present",
-    "test:browser": "NODE_ENV=test npm run test:browser --workspaces --if-present",
-    "test:node": "NODE_ENV=test npm run test:node --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
```

Comment:
Is not needed as each of internal commands should be worrying about supplying ENV flags 
fyi @danisharora099